presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -922,6 +926,7 @@ private Scope legacyVisitQuerySpecification(QuerySpecification node, Optional<Sc
             expressions.addAll(orderByExpressions);
             node.getHaving().ifPresent(expressions::add);
 
+            analyzeGroupingOperations(node, expressions, emptyList());
```

Comment:
This is leftover. I've removed it. Since new ORDER BY resolution rules are the default I don't think we need support for legacy. Let me know if you disagree.